test/integration/users.test.js

```diff
@@ -1246,6 +1246,42 @@ describe("Users", function () {
         });
     });
 
+    it("Should return users with first name 'Ankur' successfully", function (done) {
+      chai
+        .request(app)
+        .get("/users")
+        .query({ search: "Ankur", dev: true }) // Search for users with first name 'Ankur' in dev mode
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Users returned successfully!");
+          expect(res.body.users).to.be.a("array");
```

Comment:
Please least a single test that denotes that it is now searching on all 3 fields i.e. firstName, lastName, username.